docs/source/deploying-volttron/secure-deployment-considerations.rst

```diff
@@ -92,3 +92,28 @@ This approach has some limitations, including:
 - There could be cases where tampering adjusts values to incorrect but in-bounds values
   which would not be detected.
 
+
+Monitoring RabbitMQ Server
+==========================
+
+Monitoring of RabbitMQ server in deployment setup can be achieved in two ways.
+
+1. Running RabbitMQ server as a systemd service
+2. Configure VOLTTRON platform to monitor RabbitMQ server
+
+In the first case, RabbitMQ server is configured to run as a systemd service and allow systemd to
+monitor the status of the service. It can be further configured to detect and restart the RabbitMQ service
+if it crashes. VOLTTRON agents have the ability to detect when the RabbitMQ server crashes/disconnects
+and reconnect when it becomes available. In this deployment setup, a VOLTTRON platform will not
+start/stop the RabbitMQ server.
+
+In the second case, VOLTTRON configured to monitor RabbitMQ server periodically and restart if a crash is detected.
```

Comment:
Nit. Missing auxiliary verb `is`. Change to: `In the second case, VOLTTRON is configured to monitor the RabbitMQ server...`